### DISCUSSION TITLE
"Plastered" printing of articulated animals

### DISCUSSION THREAD

**Discussion by fabiovix** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#41] [General]
Hi everyone!

it's been a few days that I'm trying to print some articulated animais, like a Octopus (I work on a school, want to show the students).  But it seems to get "plastered", like too much material on the Joints when I use the PlusaSlicer 2.8.0 with the PrusaSlicer-Ender3-v3-SE-Config.
I'm using PLA, alread tested 210 and 200º degrees on the nozzle.  The strange thing is that the same STL file prints flawslessy when I slice it on the Cura 5.8.0.   Anyone had this problem?  I didn't have this problem on the Ender 3 V2 + Plusaslicer,  only on the Ender 3 V3 SE.

**Comment by suchmememanyskill** [comment]
In the cura profile, what's the flow multiplier? In my profile, it's set to 1.00x (1 mm requested = 1 mm extruded). Did you try lowering this multiplier (0.98x) or [calibrating your extruder?](https://ellis3dp.com/Print-Tuning-Guide/articles/extruder_calibration.html)

**Reply by fabiovix** [reply]
Good Morning [suchmememanyskill](https://github.com/suchmememanyskill)

Here, it's attached the gcode output from both Cura 5.8.0 and Plusa 2.8.0. 
I'm only using the default parameters from Cura for the Ender 3 V3 SE (other than I enabled the supports).  In Plusa, I'm only using this project config bundle besides enabling supports.

In CURA, regarding the flow in the "Material" tab, I'm using the default (100%). 

I'm impressed, too, about the difference in the size of the gcode generated (910kb in CURA, 14.1mb on Plusa).
[Octopus_in_Cura_and_Plusa.zip](https://github.com/user-attachments/files/16784296/Octopus_in_Cura_and_Plusa.zip)

**Comment by fabiovix** [comment]
Hi!

[suchmememanyskill](https://github.com/suchmememanyskill) I've printed on both software a small piece, for testing  support on both software.  Here are the photos.

The PlusaSlicer version is very difficult to remove the support.


![Cura](https://github.com/user-attachments/assets/b99a9bd4-a2d6-4aa3-b07d-ce2c42a63000)
![Plusa](https://github.com/user-attachments/assets/b9a1de03-7541-4e03-8b8d-695e80845e9e)
![Cura x Plusa](https://github.com/user-attachments/assets/eed2946b-9ab1-453b-b968-0f684ea7c7ab)

**Reply by suchmememanyskill** [reply]
Looks like snug is enabled in cura but not in prusaslicer, hence the difference in shape.

The contact distance is quite close (0.15mm times 2), try increasing it

**Comment by fabiovix** [comment]
Hi!  Really, it solved the problem. In my opinion, this type of support (snug), should be the default on PrusaSlicer.
